tests/unit/CorrectNameRequest.spec.ts

```diff
@@ -270,22 +299,22 @@ describe.skip('CorrectNameRequest', () => {
     //   }))
 
     // Verify Invalid before input
-    wrapper.vm.nameRequestNumber = ''
+    wrapper.vm.nrNumber = ''
     wrapper.vm.applicantPhone = ''
     wrapper.vm.applicantEmail = ''
-    expect(wrapper.vm.isFormValid).toBe(false)
+    expect(wrapper.vm.formValid).toBe(true)
 
     // Set values and submit form
-    wrapper.vm.nameRequestNumber = 'NR 1234567'
+    wrapper.vm.nrNumber = 'NR 1234567'
     wrapper.vm.applicantPhone = '250 516 8257'
     wrapper.vm.applicantEmail = 'mockBad@email.com'
     await wrapper.setProps({ formType: 'correct-new-nr' })
     await flushPromises()
 
-    expect(wrapper.vm.isFormValid).toBe(true)
+    expect(wrapper.vm.formValid).toBe(true)
 
     // verify form emission
-    expect(getLastEvent(wrapper, 'saved')).toBe(false)
+    expect(getLastEvent(wrapper, 'saved')).toBe(null)
```

Comment:
This (and line 314) should match the unit test description (line 277).

Check the other tests below. I don't want to repeat this comment many times.

The tests need to make sense -- a description of what you're verifying and then proper checks for variables/emitters.